docs/docs/adrs/adr-018-permissionless-ics.md

```diff
@@ -0,0 +1,347 @@
+
+## Decision
+In Permissionless ICS, launching an Opt In chain is **only** possible through a transaction and not through a [`MsgConsumerAddition`](https://github.com/cosmos/interchain-security/blob/v5.1.0/proto/interchain_security/ccv/provider/v1/tx.proto#L111)
+proposal. Naturally, Permissionless ICS does not eliminate governance proposals, as proposals are still necessary for Top N chains.
+Because of this, this ADR outlines a solution that also refactors governance proposals (i.e., `MsgConsumerAddition`, `MsgConsumerModification`, and `MsgConsumerRemoval`)
+so that Top N and Opt In chains can share as much functionality as possible.
+Note, that to make the distinction between governance-proposed versus transaction-launched chains clearer, in Permissionless ICS,
+we can only launch, update, or stop Top N chains with governance proposals, and we can only launch, update, or stop Opt In chains with transactions.
+Additionally, a Top N chain can transform to an Opt In chain through a gov proposal, but for simplicity, in this first
+iteration of Permissionless, an Opt In chain cannot transform to a Top N chain.
+
+### The Phases of a Consumer Chain
+
+We first present the notion of an _owner_ of a consumer chain before showing the specific phases of a consumer chain.
+
+**Owner.** A consumer chain has an _owner_, which is simply an address. Only the owner can interact (i.e., launch, update, or stop)
+with the chain. The owner of an Opt In chain is the one who signed the initial transaction to register a consumer chain (more on this later).
+Naturally, an Opt In chain can change its owner at any point. The owner of a Top N chain is the account of the governance module.
+Therefore, any changes on a Top N chain have to go through governance proposals.
+
+A consumer chain can reside in four phases: i) _registered_, ii) _initialized_, iii) _launched_, and iv) _stopped_ phase as seen
+in the diagram below:
+![Phases of a consumer chain](./adr18_phases_of_a_consumer_chain.png)
+
+**Registered phase.** In the _registered phase_, a consumer chain is assigned a unique identifier that identifies a consumer chain
+that can later be used to interact with the specific consumer chain (e.g., when a validator opts in on a chain).
+After a chain has been registered, it can later be initialized and then launched. Specifically, Permissionless ICS introduces
+a `MsgRegisterConsumer` message that can be used to register **both** Top N and Opt In consumer chains.
+In the registered phase, it is not yet known if the consumer chain would end up being a Top N or an Opt In chain and hence
+the owner of the consumer chain at this phase is the one that signed the `MsgRegisterConsumer`.
+Note that currently, a consumer chain is registered when first proposed through a `MsgConsumerAddition` proposal
+message but with Permissionless ICS, the `MsgConsumerAddition` is deprecated and chains have to issue a `MsgRegisterConsumer` message instead.
```

Comment:
We need to make sure we migrate any existing `ConsumerAddtionProposal` when we upgrade.

Reply:
Good point. Added a note in the `Additional Modifications` section.